src/tests/krate/publish/snapshots/all__krate__publish__dependencies__invalid_dependency_name.snap

```diff
@@ -5,7 +5,7 @@ expression: response.into_json()
 {
   "errors": [
     {
-      "detail": "\"🦀\" is an invalid dependency name (dependency names must start with a letter, contain only letters, numbers, hyphens, or underscores and have at most 64 characters)"
+      "detail": "invalid character `🦀` in crate name: `🦀`, the first character must be an ASCII character"
```

Comment:
previously the error message was warning about a dependency name, but now it is only saying "crate name". maybe we need another error wrapper to give more context in this case?

Reply:
Oh, nice catch. I will update it.

Reply:
I submitted https://github.com/rust-lang/crates.io/pull/7603/commits/ccbcaee3aaf5646b5bb39103b39bf353ba99c131

I don't know if this is a good way to do it. Maybe it just introduces more confusion.

Do you have any suggestions to improve it? Or maybe we can keep it as the `crate` name. Because it literally is a crate name.